### PR TITLE
Solutions for week08

### PR DESCRIPTION
**Pull request by imShafin**

### CODE REVIEW

File: week08/Count_Primes.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/// TC: O(sqrt(n) * log(n))
```
Comment (Sadman007) [inline week08/Count_Primes.cpp:1]: Where did you get that?!

File: week08/Factorial_Trailing_Zeroes.cpp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/// TC: O(Nlog5(N)) /// N is the parameter of the program
```
Comment (Sadman007) [inline week08/Factorial_Trailing_Zeroes.cpp:1]: How did you derive that?

File: week08/Pow(x, n).cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/// TC: O(logN) N is the power of the operation
+/// MC: O(logH) H is height of recursion
```
Comment (Sadman007) [inline week08/Pow(x, n).cpp:2]: What is `HEIGHT` of recursion? It can be depth.

**Review by Sadman007 — APPROVED** [review]
Not a pleasant PR that I have gone through. You need to focus. @imShafin

File: week08/Factorial_Trailing_Zeroes.cpp (hunk 2 above)
Comment (imShafin) [reply week08/Factorial_Trailing_Zeroes.cpp:1]: Is it almost O(N)?

**Pull request merged**